settings.gradle

```diff
@@ -1 +1,2 @@
-include ':library'
+include ':library', ':app'
+project(':app').projectDir = new File('example/app')
```

Comment:
This I would prefer not to merge. The example is excluded from the main gradle project on purpose. 
Could you remove it from the PR?